### PR TITLE
Merge all platform specific files into one file

### PR DESCRIPTION
**Pull request by llxia**
- PLATFORM_MAP contains all platform specific information (i.e., SPEC
and LABEL)
- Rename JENKINS_FILE to PLATFORM and remove prepended openjdk_ in the
original JENKINS_FILE value. For example, PLATFORM value is x86-64_linux
instead of openjdk_x86-64_linux
- Original PLATFORM parameter is removed

Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
- This PR updates the` testJobTemplate` file. This file needs to be approved in Adopt and Openj9 Jenkins.
- All AdoptOpenJDK jobs need to be regenerated to reflect the change.
- Once this change is merged and stabilized, all platform specific files will be removed.

File: buildenv/jenkins/testJobTemplate [hunk 1]
```diff
@@ -126,7 +126,7 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							stringParam('OPENJ9_SHA', "", "OpenJ9 sha")
 							stringParam('JDK_VERSION', "${JDK_VERSION}", "JDK version. i.e., 8, 11")
 							stringParam('JDK_IMPL', JDK_IMPL, "JDK implementation, e.g. hotspot, openj9, sap")
-							stringParam('JenkinsFile', "openjdk_${ARCH_OS}", "Jenkins file to be used")
+							stringParam('PLATFORM', "${ARCH_OS}", "Platfrom to be tested")
```
Comment (smlambert) [inline buildenv/jenkins/testJobTemplate]: typo, Platfrom -> Platform

**Comment by llxia** [comment]
Grinder links:
- x86-64_linux JDK8 openj9 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1021/
- x86-32_linux JDK11 openj9 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1022/
- x86-64_linux JDK11 hotspot nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1023/
- ppc64_aix JDK11 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1024/
- ppc64le_linux JDK8 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1025/
- s390x_linux JDK8 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1028/
- x86-64_linux_xl JDK8 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1027/
- x86-64_mac JDK8 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1029/
- x86-64_mac_xl JDK8 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1030/
- x86-64_windows JDK8 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1031/
- x86-64_windows_xl JDK8 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1032/
- x86-32_windows JDK8 nightly: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1033/
- s390x_linux JDK8 release: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1034/
- x86-64_linux_xl JDK11 release: https://ci.adoptopenjdk.net/job/Grinder_Sandbox/1036/
- Test_Job_Auto_Gen_Sandbox/21/

File: buildenv/jenkins/testJobTemplate (hunk 1 above)
Comment (llxia) [reply buildenv/jenkins/testJobTemplate]: fixed. Thanks

**Pull request merged**